pkg/controller/acicontainersoperator.go

```diff
@@ -329,6 +353,35 @@ func NewAciContainersOperator(
 			}
 		},
 	})
+
+	config_map_informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			log.Info("In AddFunc for Config-Map with key: ", key)
```

Comment:
can we put the name of the configmap in the log message? If someone reads this log message in the log, he has to go back to code and check which configmap we are talking about...

Reply:
This can add an extra line of code to fetch the name of the config-map. It can be done if really needed. However in subsequent call in handler we mention which config-map